codespell_lib/data/dictionary.txt

```diff
@@ -9939,6 +9941,8 @@ intruments->instruments
 intrusted->entrusted
 intstead->instead
 intuative->intuitive
+inturpratasion->interpretation
+inturprett->interpret
```

Comment:
Add the tion version of this misspelling.

Reply:
What?

Reply:
inturpratation->interpretation

Reply:
Sorry I don't think this should suggest inturprat, especially given it's a typo!